setup_zeromq.sh

```diff
@@ -0,0 +1,6 @@
+brew install zeromq
```

Comment:
`brew install` only works on Mac OSx.
Please change to RPi ways or a more general ways.